csharp-advanced-topics/ActionFuncDelegates/ActionAndFuncDelegatesTest/ActionFuncDelegates_Tests.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework.Internal;
+using ActionAndFuncDelegates;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+namespace ActionAndFuncDelegatesTest
+{
+    [TestClass]
+    public class ActionFuncDelegates_Tests
+    {
+        [TestMethod]
+        public void TestFunc()
```

Comment:
Please use the naming convention from the guidelines. It's strange at the beginning, but you'll get used to it.